packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+    },
+  ],
+  // TODO match or update spec https://github.com/Agoric/agoric-sdk/issues/7837
+  vaultsActual: [
+    {
+      locked: 3.425146,
+    },
+    {
+      locked: 3.0779,
+    },
+    {
+      locked: 2.846403,
+    },
+  ],
+});
+//#endregion
+
+const scale6 = x => BigInt(Math.round(x * 1_000_000));
+const DebtLimitValue = scale6(100_000);
+
+const likePayouts = ({ Bid, Collateral }) => ({
+  Collateral: {
+    value: scale6(Collateral),
+  },
+  Bid: {
+    value: scale6(Bid),
+  },
+});
+
+const makeDefaultTestContext = async t => {
+  console.time('DefaultTestContext');
+  const swingsetTestKit = await makeSwingsetTestKit(t, 'bundles/vaults', {
+    configSpecifier: '@agoric/vats/decentral-main-vaults-config.json',
+  });
+
+  const { runUtils, storage } = swingsetTestKit;
+  console.timeLog('DefaultTestContext', 'swingsetTestKit');
+  const { EV } = runUtils;
+
+  // Wait for ATOM to make it into agoricNames
+  await EV.vat('bootstrap').consumeItem('vaultFactoryKit');
+  console.timeLog('DefaultTestContext', 'vaultFactoryKit');
+
+  // has to be late enough for agoricNames data to have been published
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(
+    swingsetTestKit.storage,
+  );
+  agoricNamesRemotes.brand.ATOM || Fail`ATOM missing from agoricNames`;
+  console.timeLog('DefaultTestContext', 'agoricNamesRemotes');
+
+  const walletFactoryDriver = await makeWalletFactoryDriver(
+    runUtils,
+    storage,
+    agoricNamesRemotes,
+  );
+  console.timeLog('DefaultTestContext', 'walletFactoryDriver');
+
+  const governanceDriver = await makeGovernanceDriver(
+    swingsetTestKit,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    // TODO read from the config file
+    [
+      'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
+      'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
+      'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
+    ],
+  );
+  console.timeLog('DefaultTestContext', 'governanceDriver');
+
+  const priceFeedDriver = await makePriceFeedDriver(
+    storage,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    // TODO read from the config file
+    [
+      'agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr',
+      'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8',
+      'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
+      'agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p',
+      'agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj',
+    ],
+  );
+
+  console.timeLog('DefaultTestContext', 'priceFeedDriver');
+
+  console.timeEnd('DefaultTestContext');
+
+  return {
+    ...swingsetTestKit,
+    agoricNamesRemotes,
+    governanceDriver,
+    priceFeedDriver,
+    walletFactoryDriver,
+  };
+};
+
+test.before(async t => {
+  t.context = await makeDefaultTestContext(t);
+});
+test.after.always(t => {
+  return t.context.shutdown && t.context.shutdown();
+});
+
+// Reference: Flow 1 from https://github.com/Agoric/agoric-sdk/issues/7123
+test.serial('scenario: Flow 1', async t => {
```

Comment:
This test is currently very noisy; too noisy for me to make much sense of what's printed.